src/Filter/Filter.php

```diff
@@ -67,6 +67,7 @@ public function isActive()
      * @return string[]
      *
      * @phpstan-param array{type?: string|int|null, value?: mixed} $data
```

Comment:
```suggestion
     * @phpstan-param array{type?: int|null, value?: mixed} $data
```
Since it was updated on sonataAdmin.